src/components/lightbox/LightBox.tsx

```diff
@@ -0,0 +1,50 @@
+import { forwardRef, ReactNode } from 'react';
+
+import { IAminoTheme } from 'src/types/IAminoTheme';
+
+import { BaseDialog } from '../dialog/_BaseDialog';
+
+export type LightBoxProps = {
+  className?: string;
+  children: ReactNode;
+  onClose: () => void;
+  open: boolean;
+  theme?: IAminoTheme;
+  width?: number;
+  noBorder?: boolean;
+  /** Close when clicking outside dialog (on the backdrop)
+   * @default true
+   */
+  closeOnBlur?: boolean;
+  /** Close on pressing 'escape' key
+   * @default true
+   */
+  closeOnEsc?: boolean;
+};
+
+export const LightBox = forwardRef<HTMLDivElement, LightBoxProps>(
+  ({
+    children,
+    className,
+    onClose,
+    open,
+    theme: _theme,
+    width,
+    noBorder = true,
+    closeOnBlur,
```

Comment:
What do you think about change the prop to have an opposite meaning?
```suggestion
    width,
    withBorder = false,
    closeOnBlur,
```
With this, if you use the component, you just need to add 1 prop `withBorder` instead of `noBorder={false}`
```
<Lightbox withBorder />
```